apps/docs/content/troubleshooting/monitor-supavisor-postgres-connections.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title = "How to monitor Postgres and Supavisor connections"
+topics = [
+"database",
+"supavisor",
+]
+keywords = [
+"Grafana",
+"connections",
+"performance",
+"pooler",
+]
+
+[[errors]]
+http_status_code = 404
+code = "not_found"
+message = "The resource was not found"
+
+[[errors]]
+http_status_code = 403
+code = "forbidden"
+message = "Not allowed to access this resource"
+
+[[errors]]
+http_status_code = 500
+code = "internal_server_error"
+message = "Internal server error"
```

Comment:
Gonna remove these later as none of them are actually relevant to the issue, just needed some dummy data to see how the display looks